modules/index-service/src/main/java/org/opencastproject/index/service/impl/index/event/EventUtils.java

```diff
@@ -159,7 +159,8 @@ public static MetadataCollection getEventMetadata(Event event, EventCatalogUIAda
       }
     }
 
-    if (event.getDuration() != null) {
+    // The duration event details metadata is an optional display field
```

Comment:
Whether this field is optional or not depends on the metadata configuration...

Reply:
I can remove the comment to avoid confusion.